terraform/modules/configs/deploy-all/main.tf

```diff
@@ -607,4 +615,6 @@ module "s3_virus_scan" {
   deployment_maximum_percent         = var.deployment_maximum_percent
   deployment_minimum_healthy_percent = var.deployment_minimum_healthy_percent
   security_groups                    = [aws_security_group.s3-virus-scan.id]
+  aws_access_key_id                  = data.aws_ssm_parameter.aws_access_key_id.value
+  aws_secret_access_key              = data.aws_ssm_parameter.aws_secret_access_key.value
```

Comment:
For container secrets, I think you will need to pass the ARN here not the value, i.e. `data.aws_ssm_parameter.aws_access_key_id.arn`, `data.aws_ssm_parameter.aws_secret_access_key.arn`